api/temporalcloudapi/namespace/v1/message.pb.go

```diff
@@ -41,23 +41,23 @@ const (
 )
 
 var NamespaceSpec_SearchAttributeType_name = map[int32]string{
-	0: "Unspecified",
```

Comment:
this won't break CLI, right? CLI i think translates user provided flag values into the enum values.

Reply:
as discussed, removed these changes.